src/main/java/org/mskcc/oncokb/curation/service/MainService.java

```diff
@@ -110,6 +140,7 @@ public void annotateAlteration(Alteration alteration) {
             alteration.setGenes(pcAlteration.getGenes());
         }
         alteration.setAlteration(pcAlteration.getAlteration());
+        alteration.setName(pcAlteration.getName());
```

Comment:
Line 197 has duplicate logic with a check. Would remove one or the other depending how confident you are alteration name won't be null.

Reply:
That's to use protein change as name.